esmvaltool/cmorizers/obs/cmorize_obs_OSI_409_nh.py

```diff
@@ -0,0 +1,27 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for OSI-SAF data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   http://osisaf.met.no/p/ice/
+
+Last access
+   20190502
+
+Download and processing instructions
+    Download the desired years from the following ftp:
+        ftp://osisaf.met.no/reprocessed/ice/conc/v2p0
+    Please, keep folder structure and uncompress gz files before launching.
+
+"""
+from .utilities import read_cmor_config
+from .osi_common import cmorize_osi
+
+
+def cmorization(in_dir, out_dir):
```

Comment:
Note that the function arguments have changed since you created this pull request